File: CS50/mario.c
#include <stdio.h>
#include <cs50.c>

int main(int argc, string argv[]){

    // Declare variable n
    int n;

    // Prompt the user for a value between 1 and 8
    do
    {
        n = get_int("Height: ");
    } while (n < 1 || n > 8);

    // Create n number of rows
    for(int i = 0; i < n; i++){

        // Create spaces for drawing a pyramid
        for(int j = n-1; j > i; j--){
            printf(" ");
        }

        // Draw left side of the pyramid
        for(int k = 0; k <= i; k++){
            printf("#");
        }

        // Draw the space between the to halves of the pyramid
        printf(" ");

        // Draw the right side of the pyramid
        for(int l = 0; l <= i; l++){
            printf("#");
        }
        
        // Create a new line
        printf("\n");
    }
}

File: CS50/sentimental-mario-more/mario.py
# Promot user for integer input
while True:
    try:
        n = int(input("Height: "))
    # If input is not an integer print error message
    except ValueError:
        print("Height must be an integer between 1 to 8")
    # If integer is between 1 and 8 exit loop
    else:
        if n < 9 and n > 0:
            break

# Print n number of columns
for i in range(n):
    # Print spaces for left side of pyramid
    for j in range(n - 1, i, -1):
        print(" ", end="")
    # Print left side of pyramid
    for k in range(i + 1):
        print("#", end="")
    # Add double space between the two sides
    print("  ", end="")
    # Print right side of pyramid
    for l in range(i + 1):
        print("#", end="")
    print()